src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -517,6 +533,9 @@ private void ScanInterpolatedStringLiteralEnd(InterpolatedStringKind kind, int s
                 {
                     Debug.Assert(kind is InterpolatedStringKind.SingleLineRaw or InterpolatedStringKind.MultiLineRaw);
                     ScanRawInterpolatedStringLiteralEnd(kind, startingQuoteCount);
+
+                    if (!_isInterpolatedString)
+                        _lexer.ScanUtf8Suffix();
```

Comment:
This is needed because non-interpolated raw strings can now go through this path, right?

Reply:
Correct